### DISCUSSION TITLE
Changing the DNS provider

### DISCUSSION THREAD

**Discussion by vladik-vitre** [polymind-inc/acmebot#713] [Show and tell]
We had an issue with GoDaddy API which doesn't allow ACME certificate renewal anymore.
We moved to an Azure DNS zone (not private) and had an issue renewing our existing certificates in the Key Vault after doing all needed configuration.
After going over the function, files and stuff, we went to the source code and discovered that the certificate has tags on it which tell what DNS provider is being used.
We changed the tag from "GoDaddy" to "Azure DNS" and it sorted out the issue.
Didn't find any discussions about this need and no issues either so if anyone needs this information, please let me know and I'll try to help.

Good luck.

**Comment by vladik-vitre** [comment]
If it wasn't clear, we left the DNS registrar on GoDaddy but changed the nameserver addresses to be the ones on Azure DNS.
Now we make changes only on Azure of course, and we use the Azure DNS provider which works with ACME.

**Comment by anthonyvia** [comment]
Thank you for this! We just needed to do this.
